tests/CommentsGridFieldActionTest.php

```diff
@@ -40,6 +41,9 @@ class CommentsGridFieldActionTest extends SapphireTest
     protected function setUp(): void
     {
         parent::setUp();
+        if (Deprecation::isEnabled()) {
+            $this->markTestSkipped('Test calls deprecated code');
+        }
         $this->list = new DataList(Team::class);
         $config = CommentsGridFieldConfig::create()->addComponent(new GridFieldDeleteAction());
         $this->gridField = new CommentsGridField('testfield', 'testfield', $this->list, $config);
```

Comment:
Should we leave `setUp` method to `$gridField ` initialisation?
And mark as deprecated `$form` https://github.com/silverstripe/silverstripe-comments/blob/b771bb624afc8c8deaa368883a6044e331f1a2e0/tests/CommentsGridFieldActionTest.php#L50

Reply:
It's a unit test, no need to deprecate things